css/cards.css

```diff
@@ -0,0 +1,117 @@
+body {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
don't override tag selectors (e.i. `body`, `main`, `header`, etc.) from one stylesheet to another. It may cause conflicts in further steps